egs/librispeech/s5/local/chain/run_tdnn_6z.sh

```diff
@@ -0,0 +1,208 @@
+set -e
+
+# configs for 'chain'
+affix=
+stage=11
+train_stage=-10
+speed_perturb=true
+get_egs_stage=-10
+dir=exp/chain/tdnn_6z  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+
+# TDNN options
+# this script uses the new tdnn config generator so it needs a final 0 to reflect that the final layer input has no splicing
+# smoothing options
+pool_window=
+pool_type='none'
+pool_lpfilter_width=
+self_repair_scale=0.00001
+# training options
+num_epochs=4
+initial_effective_lrate=0.001
+final_effective_lrate=0.0001
+leftmost_questions_truncate=-1
+max_param_change=2.0
+final_layer_normalize_target=0.5
+num_jobs_initial=3
+num_jobs_final=16
+minibatch_size=128
+relu_dim=576
```

Comment:
No. I used 725.

在 2016/4/5 1:45, Vijayaditya Peddinti 写道:
>
> In egs/librispeech/s5/local/chain/run_tdnn_6z.sh 
> <https://github.com/kaldi-asr/kaldi/pull/645#discussion_r58416499>:
>
> > +# smoothing options
> > +pool_window=
> > +pool_type='none'
> > +pool_lpfilter_width=
> > +self_repair_scale=0.00001
> > +# training options
> > +num_epochs=4
> > +initial_effective_lrate=0.001
> > +final_effective_lrate=0.0001
> > +leftmost_questions_truncate=-1
> > +max_param_change=2.0
> > +final_layer_normalize_target=0.5
> > +num_jobs_initial=3
> > +num_jobs_final=16
> > +minibatch_size=128
> > +relu_dim=576
>
> I think you can further increase this. Is this the value @naxingyu 
> <https://github.com/naxingyu> uses in SWBD+Fisher recipe ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/645/files/9ea0ff3e250cec5ed8f22b1a837c17791398eaf6#r58416499>
>